### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by michaelck** [CityOfPhiladelphia/property2#176]
Running the page through the [Web Accessibility Evaluation Tool](http://wave.webaim.org/) reveals a number of errors. Most of these can be addressed relatively quickly:
- [x] Document language missing ( add `lang="en"` to opening `html` element )
- [x] Linked logo image needs `alt` attribute. Typically this can be left as `alt=""` for decorative images however logos typically receive alt text . Consider giving the link an `aria-label`
- [x] The input fields are missing labels. Important for those visitors who use screen readers, these can be added without altering the look of the page by using the `show-for-sr` class
- [x] Submit button `input` elements require a `value`
- [x] Broken skip link
- [x] Broken `aria-labelledby` on ToS modal
- [x] Results pages have some heading order issues

**Comment by michaelck** [comment]
Started a branch at [a11y](https://github.com/CityOfPhiladelphia/property2/tree/a11y). Checked off the items it's addressed so far.
